pytest_html/plugin.py

```diff
@@ -213,9 +213,15 @@ def append_extra_html(self, extra, extra_index, test_index):
         def append_log_html(self, report, additional_html):
             log = html.div(class_='log')
             if report.longrepr:
-                for line in str(report.longrepr).splitlines():
+                if hasattr(report, 'longreprtext'):
```

Comment:
There was and there is no problem with `longreprtext` IF it is available. But on travis there were failures with pytest29 so I made a check for it and if it is not available made a different version for python 3 and python 2 and it seemed ok until checking results on travis where there left problem with flake8 on python3. And this was last problem to solve to have tests passing. I think it is a fight to overcome problem in flake8 itself if I correctly understand what it does. That warning is in code that is not executed on python3.